closed/Intel/code/README.md

```diff
@@ -568,8 +569,9 @@ bash run_calibration.sh
 ### Run the Benchmark
 
 ```
-export DATA_DIR=${WORKLOAD_DATA}/openimages
-export MODEL_PATH=${WORKLOAD_DATA}/retinanet-int8-model.pth
+export DATA_DIR=data/openimages

```

Comment:
${WORKLOAD_ATA} with absolute path looks good to me. why we use relative path "data" instead?

Reply:
Right. The WORKLOAD_DATA was not set in the container prior to these exports. I'll set it, and revert to using the original.

Reply:
Added WORKLOAD_DATA prior to the exports.